### ISSUE TITLE
InvalidTemplateLibrary at /route/

### ISSUE THREAD

**Issue by kiruh** [bartTC/django-memcache-status#19]
Invalid template library specified. ImportError raised when trying to load 'memcache_status.templatetags.memcache_status_tags': cannot import name 'get_cache'.

I found where error is coming from.
debug_toolbar.panels.cache has neither base_get_cache nor get_cache methods.

```
try:
    from django.core.cache import caches
except ImportError:
    from django.core.cache import get_cache as caches

if caches.__module__.startswith('debug_toolbar'):
    try:
        from debug_toolbar.panels.cache import base_get_cache as caches
    except ImportError:
--->    from debug_toolbar.panels.cache import get_cache as caches
```
I've tried to remove imports from debug_toolbar and it works as expected. So why get_cache should be imported from debug_toolbar.panels.cache. Is there any reason?

Django==2.1.7
django-debug-toolbar==1.11
django-memcache-status==2.0.1

**Comment by bartTC** [comment]
> So why get_cache should be imported from debug_toolbar.panels.cache. Is there any reason?

🤔 Mmh good point. 10 years ago I built a Cache panel for debug toolbar with this but thats not used anymore.

What memcache bindings did you use?

**Comment by bartTC** [comment]
#5 is also related. 

> When using django-debug-toolbar with django-memcache-status, nothing shows up in admin since debug toolbar wraps cache with proxy object with no _cache field. Fixed.

**Comment by kiruh** [comment]
> What memcache bindings did you use?

CACHES = {
    "default": {
        "BACKEND": "django.core.cache.backends.memcached.MemcachedCache",
        "LOCATION": "127.0.0.1:11211",
    }
}

python-memcached==1.59

**Comment by kiruh** [comment]
#20 has fixed this problem for me

**Comment by bartTC** [comment]
I tried it myself with your versions above and everything is working fine for me. 

Django==2.1.7
django-debug-toolbar==1.11
django-memcache-status==2.0.1
python-memcached==1.59

However I also figured that the whole `caches.__module__` thing does not seem to be necessary anymore so I'd rather remove it comletely which should also fix your problem. I'm updating the tests a bit and will then provide a release.
